Jenkinsfile

```
@@ -0,0 +1,54 @@
+properties([pipelineTriggers([githubPush()])])
+pipeline {
+  options {
+    disableConcurrentBuilds()
+    buildDiscarder(logRotator(numToKeepStr: '3'))
+    timeout(time: 30, unit: 'MINUTES')
+  }
+
+  agent {
+    kubernetes {
+      label 'worker-docker'
+      inheritFrom 'default'
+
+      containerTemplates([
+        containerTemplate(name: 'golang', image: 'golang:1.12.5', ttyEnabled: true, command: 'cat')
```

go 1.12 is very old, let's use the latest go 1.x

```suggestion
        containerTemplate(name: 'golang', image: 'golang:1', ttyEnabled: true, command: 'cat')
```